crates/ruma-common/CHANGELOG.md

```diff
@@ -22,6 +22,9 @@ Breaking changes:
 * Fix the `RoomMessageEventContent::*_reply_plain` methods that now return a
   message with a `formatted_body`, according to the spec. Therefore, they only
   accept `OriginalRoomMessageEvent`s like their HTML counterparts.
+* Update the `state_key` field of state events to be of a different type
+  depending on the content type. You now no longer need to validate manually
+  that `m.room.member` events have a user ID as their state key!
```

Comment:
Will this cause issues with conduit validating state events received over federation? Or is this only impacting client-server APIs?

Reply:
Conduit can't really use Ruma's event types for various reasons, so this should be fine.